src/test/java/org/gridsuite/modification/server/modifications/BatteryCreationInBusBreakerTest.java

```diff
@@ -91,6 +97,7 @@ protected void assertAfterNetworkModificationCreation() {
         assertNotNull(getNetwork().getBattery("idBattery2"));
         assertEquals(1, getNetwork().getVoltageLevel("v1").getBatteryStream()
                 .filter(transformer -> transformer.getId().equals("idBattery2")).count());
+        assertEquals(PROPERTY_VALUE, getNetwork().getBattery("idBattery2").getProperty(PROPERTY_NAME));
```

Comment:
in the test utils I created a method to create a FreePropertyInfos easily you can use this one instead, ModificationCreation.getFreePropertyInfos()